core/templates/dev/head/pages/exploration_editor/editor_tab/state_editor_hints.html

```diff
@@ -92,7 +95,7 @@
 
   <div class="modal-body">
     <p>
-      Are you sure you want to delete this hint?
+      Are you sure you want to delete this hint? Please note, statistics for the entire series of hints will be reset.
```

Comment:
Are you sure you want to delete this hint? This will reset the view counts for this set of hints.